lighthouse-core/lib/i18n/en-US.json

```diff
@@ -1319,6 +1319,10 @@
     "message": "Expand snippet",
     "description": "Label for button that shows all lines of the snippet when clicked"
   },
+  "lighthouse-core/report/html/renderer/util.js | thirdPartyResourcesLabel": {
+    "message": "Show 3rd party resources",
+    "description": "This label is shown above a table next to the checkbox to hide 3rd party resources"
```

Comment:
this probably needs to improve 😄

Reply:
heh :) 

Maybe...

`Label for checkbox that toggles the visibility of "3rd party resources" in the adjacent table. "3rd party resources" are any items in the table that came from another domain that likely isn't owned or controlled by the developer of the webpage.`